src/styles/pages/_object-detail.scss

```diff
@@ -286,6 +298,7 @@ $metadata-collapsed-width: 36vw;
 		.p-object-detail__video {
 			display: grid;
 			grid-template-rows: 1fr auto;
+			margin-top: 0.1rem;
```

Comment:
We need this line to prevent this funky thingy:
<img width="1012" alt="Screenshot 2023-03-29 at 11 19 51" src="https://user-images.githubusercontent.com/77958553/228511734-f3421f91-c2f0-41ff-b619-2d3151b4dc72.png">

Reply:
Add a comment explaining the issue in the code.